### ISSUE TITLE
Leftover -I${efivar}/include/efivar CFLAGS

### ISSUE THREAD

**Issue by neilmayhew** [NixOS/nixpkgs#35011]
## Issue description

`efivar.pc` uses a prefix of `/usr/include`

### Steps to reproduce

```sh
nix-shell '<nixpkgs>' -A efibootmgr --command 'pkg-config --cflags efivar'
```

It doesn't have to be `efibootmgr`: any package that uses `efivar` will do.

I'm trying to package a utility (`mokutil`) that uses `efivar` via `pkg-config` and it can't find `<efivar.h>` because `pkg-config --cflags efivar` returns `-I/usr/include/efivar`.

`efibootmgr` builds because its derivation doesn't rely on using `pkg-config`. It uses `NIX_CFLAGS_COMPILE = "-I${efivar}/include/efivar";`.

I've added this attribute to my derivation for `mokutil` and it now builds, but it shouldn't be necessary to add this.

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.9.80, NixOS, 17.09.3008.c831224528c (Hummingbird)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 1.11.16`
 - channels(user): `""`
 - channels(root): `"nixos-17.09.3008.c831224528c, unstable-18.03pre126508.8ecadc12502"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by jtojnar** [comment]
This was fixed in #32751.

**Comment by jtojnar** [comment]
We should probably remove the hacks now:

```
$ rg include/efivar
pkgs/tools/system/efibootmgr/default.nix
18:  NIX_CFLAGS_COMPILE = "-I${efivar}/include/efivar";

pkgs/os-specific/linux/firmware/fwupdate/default.nix
31:        export NIX_CFLAGS_COMPILE="$NIX_CFLAGS_COMPILE -I${gnu-efi}/include/efi -I${efivar}/include/efivar -I${gnu-efi}/include/efi/$arch"
```

**Comment by neilmayhew** [comment]
My apologies. I tested this with a checkout of `master` that wasn't up to date.

**Comment by neilmayhew** [comment]
Actually, I should leave this open because of the need for cleanup that was mentioned.

**Comment by jtojnar** [comment]
Fixed in #35065
